docs/run.md

```diff
@@ -128,11 +128,13 @@ https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage):
 
 4. Run LISA
 
-    Use above `<subscription id>` and `<private key file>` to replace in below command. It may take several minutes to complete.
+    Use above `<subscription id>` and `<private key file>` to replace in below command. It may take several minutes to complete. Gallery images can be found according to the instructions [here](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage)
 
     ```bash
-    lisa -r ./microsoft/runbook/azure.yml -v subscription_id:<subscription id> -v "admin_private_key_file:<private key file>"
+    lisa -r ./microsoft/runbook/azure.yml -v subscription_id:<subscription id> -v "admin_private_key_file:<private key file>" -v "gallery_image:<gallery image string>"
     ```
+  
```

Comment:
```
  environments:
    - nodes:
        - type: requirement
          core_count:
            min: 2
          azure:
            marketplace: $(marketplace_image)
            location: $(location)

```

Reply:
BTW we'de better fix the extra indentation in that file :)

Reply:
This is weird, i'm on main adn the example azure.yml has different names in it.
`  environments:
    - nodes:
        - type: requirement
          core_count:
            min: 2
          azure:
            gallery: $(gallery_image)
            location: $(location)
        - type: requirement
          core_count:
            min: 2
          azure:
            gallery: $(gallery_image)
            location: $(location)`

Reply:
https://github.com/microsoft/lisa/blob/main/microsoft/runbook/azure.yml this?